src/generateNumberCards.js

```diff
@@ -0,0 +1,25 @@
+function randomInt(max, min) {
+    return Math.floor(Math.random() * (max - min + 1) + min)
+}
+
+function createMultipleCardsBetween3And5(){
+let randomNumbersHolder = {}
+    for (let i = 0; i < randomInt(5,3); i++) {
+        let randomNumber = randomInt(100, 0)
+        while (Object.values(randomNumbersHolder).includes(randomNumber)) {
+            randomNumber = randomInt(100, 0)
```

Comment:
Since you already set the variable, you may not need the '=randomInt(100,0)' when executing the loop. Not sure though

Reply:
I think we need it for the while loop to work